naughty/fedora-41/7631-stratis-crypto-pool-boot

```diff
@@ -0,0 +1,3 @@
+Traceback (most recent call last):
+  File "check-storage-stratis", line *, in testEncrypted
```

Comment:
is this going to match the `File "/work/make-checkout-workdir/test/verify/check-storage-stratis", line 680, in testEncrypted` ?

to be specific I mean the file path to the test. taken from the [test run](https://cockpit-logs.us-east-1.linodeobjects.com/pull-21895-8d718c8b-20250408-085420-fedora-42-storage/log.html)

Reply:
Yes, we do some normalization on file names, see https://github.com/cockpit-project/bots/blob/b1aea4fbbb29e2a816af95b38b74749d22c8d7bd/test-failure-policy#L143